src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1416,6 +1423,20 @@ private void InitializeCustomNodeManager()
                 customNodeSearchRegistry.Add(info.FunctionId);
                 var searchElement = new CustomNodeSearchElement(CustomNodeManager, info);
                 SearchModel.Add(searchElement);
+
+                //Indexing node packages installed using PackageManagerSearch
+                var iDoc = LuceneSearchUtility.InitializeIndexDocumentForNodes();
+                if (searchElement != null)
+                {
+                    AddNodeTypeToSearchIndex(searchElement, iDoc);
+                }
+
+                if (!IsTestMode)
+                {
+                    //Commit the packages node info indexed
+                    LuceneSearchUtility.writer?.Commit();
+                }
```

Comment:
same comment here

Reply:
Fixed in commit: https://github.com/DynamoDS/Dynamo/pull/14120/commits/85334fcdf0a658d74b5692fdad74005b7c6978a2